### DISCUSSION TITLE
How to deal with Big Energy RMSE?

### DISCUSSION THREAD

**Discussion by haitaier** [deepmodeling/deepmd-kit#3788] [Q&A]
I have created a small system (the number of atoms is 10 in total)for practising. 
AIMD calculations converge pretty well,Is there something wrong with my json file?
The system has a target accuracy of e-4 for energy and e-2 for force.
Now,my results show that the accuracy of the force is up to snuff, but the energy is not accurate enough.Which parameters are set unreasonably?
Thank You.
Here is json:
{
    "_comment": " model parameters",
    "model": {											
	"type_map":	["Na", "Cl"],									
	"descriptor" :{										 
	    "type":		"se_e2_a",							
	    "sel":		[10, 10],							
	    "rcut_smth":	0.50,							
	    "rcut":		6.00,									
	    "neuron":		[20, 40, 80],					
	    "resnet_dt":	false,							
	    "axis_neuron":	4,								
	    "seed":		1,
	    "_comment":		" that's all"
	}, 
	"fitting_net" : {
	    "neuron":		[200, 200, 200],				
	    "resnet_dt":	true,
	    "seed":		1,
	    "_comment":		" that's all"
	},
	"_comment":	" that's all"
    },

    "learning_rate" :{									
	"type":		"exp",									
	"decay_steps":	2000,									
	"start_lr":	0.001,									
	"stop_lr":	3.51e-8,								
	"_comment":	"that's all"
    },

    "loss" :{											
	"type":		"ener",
	"start_pref_e":	0.02,								
	"limit_pref_e":	1,									
	"start_pref_f":	1000,								
	"limit_pref_f":	1,									
	"start_pref_v":	0,									
	"limit_pref_v":	0,
	"_comment":	" that's all"
    },

    "training" : {										
	"training_data": {
	    "systems":     ["../00.data/training_data"],	
	    "batch_size":  "auto",							
	    "_comment":	   "that's all"
	},
	"validation_data":{
	    "systems":	   ["../00.data/validation_data"],	 	
	    "batch_size":  "auto",							
	    "numb_btch":   1,								
	    "_comment":	   "that's all"
	},
	"numb_steps":	400000,								
	"seed":		10,
	"disp_file":	"lcurve.out",						
	"disp_freq":	1000,								
	"save_freq":	10000,
	"_comment":	"that's all"
    },    

    "_comment":		"that's all"
}
[input.json](https://github.com/deepmodeling/deepmd-kit/files/15321508/input.json)
[lcurve.txt](https://github.com/deepmodeling/deepmd-kit/files/15321522/lcurve.txt)

**Comment by njzjz** [comment]
I feel `sel` is pretty small.

**Reply by ZKNB** [reply]
Could you take a look at it for me, I'm having a similar problem with parameterization #3783

**Reply by haitaier** [reply]
I performed the dp test and the results show that the error for energy is rather small, and the error for force is of the order of magnitude.What's the problem? Is it the AIMD part ? Or is it the modeling?
[results.e.txt](https://github.com/deepmodeling/deepmd-kit/files/15374093/results.e.txt)
[results.f .txt](https://github.com/deepmodeling/deepmd-kit/files/15374094/results.f.txt)

**Comment by chtchelkatchev** [comment]
You can try
 "sel":		"auto",
